bin/VMSGUI

```diff
@@ -47,8 +51,24 @@ class EUI(QMainWindow):
         i = 0
         for s in self.SYSTEMS:
             m = menuBar.addMenu(s)
+            m.addAction(
+                "New &raw graph",
```

Comment:
"New & raw graph"

Reply:
No. & is used to mark accelerator key - R in this case.